Tests/MapboxMapsTests/Foundation/GeoJSON/FeatureCollectionTests.swift

```diff
@@ -3,25 +3,20 @@ import CoreLocation
 @testable import MapboxMaps
 
 // Disabling rules against force casting for test file.
-// swiftlint:disable explicit_top_level_acl explicit_acl force_try force_cast
+// swiftlint:disable explicit_top_level_acl explicit_acl force_try
```

Comment:
None of these should be needed in tests anymore
```suggestion
```

Reply:
Actually, we should just delete this whole file. I don't see why we would need to test Turf in this repo. I'll ticket that separately to make sure I clean up any unused fixtures as well.